tests/integration/adapter-test.js

```diff
@@ -597,3 +597,37 @@ test('Saving a record with a quotes in a string attribute', function(assert) {
     });
   });
 });
+
+test('Saving a record with a back slash in a string attribute', function(assert) {
+  assert.expect(2);
+
+  run(function() {
+    store.push({
+      data: {
+        type: 'post',
+        id: '1',
+        attributes: {
+          name: 'Rails is omakase'
+        }
+      }
+    });
+  });
+
+  ajaxResponse({
+    data: {
+      post: {
+        id: '1',
+        name: '\\\\Ember.js is da bomb.'
+      }
+    }
+  });
```

Comment:
I think i was just confusing myself with how many \'s we would need to escape the escape, and then actually have a backslash?